src/executor/run_commands.c

```diff
@@ -58,16 +48,27 @@ int	run_multi_processes(char *env[],
 
 t_bool	is_single_command(int num_of_cmds, t_command_code command_code)
 {
-	return (num_of_cmds == 1 && command_code != SYSTEM);
+	return (num_of_cmds == 1 && (command_code == EXIT
+			|| (command_code >= EXPORT && command_code <= ENV)));
```

Comment:
So I think this condition could be simplified.

I think we can add this function to environment_commands.c:
t_bool is_env_command(t_command_code code)
{
       return (code == EXPORT || code == UNSET || code == ENV);
}
because if we add more commands or change the order we may break this, this way we avoid it.

then:
return (num_of_cmds == 1 && (command_code == EXIT || is_env_command(command_code));